src/www/ui_components/app-root.html

```diff
@@ -461,50 +466,50 @@
-            hi: {id: "hi", dir: "ltr"},
-            hr: {id: "hr", dir: "ltr"},
-            hu: {id: "hu", dir: "ltr"},
-            id: {id: "id", dir: "ltr"},
-            it: {id: "it", dir: "ltr"},
-            ja: {id: "ja", dir: "ltr"},
-            ko: {id: "ko", dir: "ltr"},
-            km: {id: "km", dir: "ltr"},
-            lt: {id: "lt", dir: "ltr"},
-            lv: {id: "lv", dir: "ltr"},
-            nl: {id: "nl", dir: "ltr"},
-            no: {id: "no", dir: "ltr"},
-            pl: {id: "pl", dir: "ltr"},
-            "pt-BR": {id: "pt-BR", dir: "ltr"},
-            ro: {id: "ro", dir: "ltr"},
-            ru: {id: "ru", dir: "ltr"},
-            sk: {id: "sk", dir: "ltr"},
-            sl: {id: "sl", dir: "ltr"},
-            sr: {id: "sr", dir: "ltr"},
-            "sr-Latn": {id: "sr-Latn", dir: "ltr"},
-            sv: {id: "sv", dir: "ltr"},
-            th: {id: "th", dir: "ltr"},
-            tr: {id: "tr", dir: "ltr"},
-            uk: {id: "uk", dir: "ltr"},
-            ur: {id: "ur", dir: "rtl"},
-            vi: {id: "vi", dir: "ltr"},
-            zh: {id: "zh", dir: "ltr"},
-            "zh-CN": {id: "zh-CN", dir: "ltr"},
-            "zh-TW": {id: "zh-TW", dir: "ltr"},
+            am: {id: "am", name: "አማርኛ", dir: "ltr"},
+            ar: {id: "ar", name: "العربية", dir: "rtl"},
+            bg: {id: "bg", name: "Български", dir: "ltr"},
+            ca: {id: "ca", name: "Català", dir: "ltr"},
+            cs: {id: "cs", name: "Česky", dir: "ltr"},
+            da: {id: "da", name: "Dansk", dir: "ltr"},
+            de: {id: "de", name: "Deutsch", dir: "ltr"},
+            el: {id: "el", name: "Ελληνικά", dir: "ltr"},
+            en: {id: "en", name: "English", dir: "ltr"},
+            "es-419": {id: "es-419", name: "Español (Latinoamérica)", dir: "ltr"},
+            fa: {id: "fa", name: "فارسی", dir: "rtl"},
+            fi: {id: "fi", name: "Suomi", dir: "ltr"},
+            fil: {id: "fil", name: "Wikang Filipino", dir: "ltr"},
+            fr: {id: "fr", name: "Français", dir: "ltr"},
+            he: {id: "he", name: "עברית", dir: "rtl"},
+            hi: {id: "hi", name: "हिन्दी", dir: "ltr"},
+            hr: {id: "hr", name: "Hrvatski", dir: "ltr"},
+            hu: {id: "hu", name: "Magyar", dir: "ltr"},
+            id: {id: "id", name: "Bahasa Indonesia", dir: "ltr"},
+            it: {id: "it", name: "Italiano", dir: "ltr"},
+            ja: {id: "ja", name: "日本語", dir: "ltr"},
+            km: {id: "km", name: "ភាសាខ្មែរ", dir: "ltr"},
+            ko: {id: "ko", name: "한국어", dir: "ltr"},
+            lt: {id: "lt", name: "Lietuvių", dir: "ltr"},
+            lv: {id: "lv", name: "Latviešu", dir: "ltr"},
+            nl: {id: "nl", name: "Nederlands", dir: "ltr"},
+            no: {id: "no", name: "Norsk (bokmål / riksmål)", dir: "ltr"},
+            pl: {id: "pl", name: "Polski", dir: "ltr"},
+            "pt-BR": {id: "pt-BR", name: "Português (Brasil)", dir: "ltr"},
```

Comment:
nit: same here - this is the only Portuguese variant we offer.

Reply:
Done